fast/stages/03-gke-multitenant/prod/main.tf

```diff
@@ -66,15 +74,62 @@ module "gke-project-0" {
   # }
 }
 
+module "gke-project-1" {
```

Comment:
I prefer having a single project, at least initially. As you can see things get more complicated if we allow more than one.